docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -281,63 +281,98 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
     * This should be ``main`` in the case of an unspecified entry point.
     * This will be the base name of the tasks file to execute, with no ``.yml`` or ``.yaml`` file extension.
 
-    :short_description:
+.. list-table:: Role Argument Specification Fields
+  :width: 90%
+  :widths: auto
+  :header-rows: 1
+
+    * - Name
+      - Description
+      - Type
+      - Required
+      - Default
```

Comment:
Tables with multiple columns tend to be a problem on narrow-width devices, since you have to scroll horizontally to see everything.